src/main/java/se/kth/sda/skeleton/posts/PostRepository.java

```diff
@@ -1,7 +1,7 @@
 package se.kth.sda.skeleton.posts;
 
-/*
-    @TODO extend the appropriate JpaRepository to get common database operations for Post
- */
+import org.springframework.stereotype.Repository;
+
+@Repository
 public interface PostRepository {
```

Comment:
should we extend the interface in this PR?

`public interface PostRepository extends JpaRepository<Post, Long> {`

Reply:
Extended the interface